### PR TITLE
added Sinhala translation of the code of conduct

### PR DESCRIPTION
**Pull request by Viraj121**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Review by samithkavishke — COMMENTED** [review]
Yeah looks good

File: docs/CODE_OF_CONDUCT-si.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+මෙම චර්යාධර්ම සංග්‍රහයට නොගැලපෙන, හෝ තාවකාලිකව තහනම් කිරීමට හෝ
+ඔවුන් නුසුදුසු යැයි සලකන වෙනත් හැසිරීම් සඳහා ස්ථිරවම ඕනෑම දායකයෙක්,
+තර්ජනාත්මක, ආක්‍රමණශීලී හෝ හානිකර.
+
+මෙම චර්යාධර්ම සංග්‍රහය අනුගමනය කිරීමෙන්, ව්‍යාපෘති නඩත්තු කරන්නන් කැපවී සිටිති
+කළමනාකරණයේ සෑම අංශයකටම සාධාරණව සහ ස්ථාවරව මෙම මූලධර්ම යෙදීම
+මෙම ව්යාපෘතිය. සංග්‍රහය අනුගමනය නොකරන හෝ බලාත්මක නොකරන ව්‍යාපෘති නඩත්තු කරන්නන්
+ව්‍යාපෘති කණ්ඩායමෙන් හැසිරීම ස්ථිරවම ඉවත් කළ හැක.
+
+මෙම චර්යාධර්ම සංග්‍රහය ව්‍යාපෘති අවකාශයන් තුළ මෙන්ම පොදු අවකාශයන්හිද අදාළ වේ
```
Comment (samithkavishke) [inline docs/CODE_OF_CONDUCT-si.md:34]: does not give the correct meaning

File: docs/CODE_OF_CONDUCT-si.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+විවෘත සහ පිළිගැනීමේ ප්‍රජාවක් පෝෂණය කරමින්, සියලු මිනිසුන්ට ගරු කිරීමට අපි ප්‍රතිඥා දෙමු
+ගැටළු වාර්තා කිරීම, විශේෂාංග ඉල්ලීම් පළ කිරීම, යාවත්කාලීන කිරීම හරහා දායක වන්න
+ලියකියවිලි, ඇදීමේ ඉල්ලීම් හෝ පැච් ඉදිරිපත් කිරීම සහ වෙනත් ක්‍රියාකාරකම්.
+
+මෙම ව්‍යාපෘතියට සහභාගී වීම හිරිහැරයකින් තොරව කිරීමට අපි කැපවී සිටිමු
+අත්දැකීම් මට්ටම, ස්ත්‍රී පුරුෂ භාවය, ස්ත්‍රී පුරුෂ භාවය නොසලකා සෑම කෙනෙකුටම අත්දැකීම්
+අනන්‍යතාවය සහ ප්‍රකාශනය, ලිංගික දිශානතිය, ආබාධිතභාවය, පුද්ගලික පෙනුම,
+ශරීර ප්‍රමාණය, ජාතිය, වාර්ගිකත්වය, වයස, ආගම හෝ ජාතිකත්වය.
+
+සහභාගිවන්නන් විසින් පිළිගත නොහැකි හැසිරීම් සඳහා උදාහරණ ඇතුළත් වේ:
```
Comment (samithkavishke) [inline docs/CODE_OF_CONDUCT-si.md:13]: state as below is the correct one, පහත දක්වා ඇත. change it

File: docs/CODE_OF_CONDUCT-si.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+මෙම චර්යාධර්ම සංග්‍රහය ව්‍යාපෘති අවකාශයන් තුළ මෙන්ම පොදු අවකාශයන්හිද අදාළ වේ
+පුද්ගලයෙකු ව්‍යාපෘතිය හෝ එහි ප්‍රජාව නියෝජනය කරන විට.
+
+අපයෝජන, හිරිහැර කිරීම හෝ වෙනත් ආකාරයකින් පිළිගත නොහැකි හැසිරීම් අවස්ථා විය හැකිය
+gmail.com හි victorfelder හි ව්‍යාපෘති නඩත්තු කරන්නෙකු සම්බන්ධ කර ගැනීමෙන් වාර්තා විය. සෑම
+පැමිණිලි සමාලෝචනය කර විමර්ශනය කරනු ලබන අතර එයට ප්‍රතිචාරයක් ලැබෙනු ඇත
+අවශ්‍ය සහ තත්වයන්ට සුදුසු යැයි සලකනු ලැබේ. නඩත්තු කරන්නන් වේ
+හි වාර්තාකරු සම්බන්ධයෙන් රහස්‍යභාවය පවත්වා ගැනීමට බැඳී සිටී
+සිද්ධිය.
+
```
Comment (samithkavishke) [inline docs/CODE_OF_CONDUCT-si.md:43]: Above paragraph is not accurate

File: docs/CODE_OF_CONDUCT-si.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
+# දායක චර්යා සංග්‍රහය
+
+මෙම ව්‍යාපෘතියේ දායකයින් සහ නඩත්තු කරන්නන් ලෙස, සහ උනන්දුව සඳහා
+විවෘත සහ පිළිගැනීමේ ප්‍රජාවක් පෝෂණය කරමින්, සියලු මිනිසුන්ට ගරු කිරීමට අපි ප්‍රතිඥා දෙමු
```
Comment (samithkavishke) [inline docs/CODE_OF_CONDUCT-si.md:4]: state all the sentence in simple Sinhala don't use a translator

File: docs/CODE_OF_CONDUCT-si.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+* පොදු හෝ පෞද්ගලික හිරිහැර
+* භෞතික හෝ විද්‍යුත් වැනි වෙනත් අයගේ පුද්ගලික තොරතුරු ප්‍රකාශයට පත් කිරීම
+   ලිපිනයන්, පැහැදිලි අවසරයකින් තොරව
+* වෙනත් සදාචාර විරෝධී හෝ වෘත්තීය නොවන හැසිරීම්
+
+ඉවත් කිරීමට, සංස්කරණය කිරීමට, හෝ කිරීමට ව්‍යාපෘති නඩත්තු කරන්නන්ට අයිතිය සහ වගකීම ඇත
+අදහස්, කැපවීම්, කේතය, විකි සංස්කරණ, ගැටළු සහ වෙනත් දායකත්වයන් ප්‍රතික්ෂේප කරන්න
+මෙම චර්යාධර්ම සංග්‍රහයට නොගැලපෙන, හෝ තාවකාලිකව තහනම් කිරීමට හෝ
+ඔවුන් නුසුදුසු යැයි සලකන වෙනත් හැසිරීම් සඳහා ස්ථිරවම ඕනෑම දායකයෙක්,
+තර්ජනාත්මක, ආක්‍රමණශීලී හෝ හානිකර.
```
Comment (samithkavishke) [inline docs/CODE_OF_CONDUCT-si.md:27]: Not correct in grammar

File: docs/CODE_OF_CONDUCT-si.md (hunk 1 above)
Comment (Viraj121) [reply docs/CODE_OF_CONDUCT-si.md:34]: okay I'll look into it

**Review by samithkavishke — CHANGES_REQUESTED** [review]

**Comment by eshellman** [comment]
thank you @samithkavishke for helping with review.

**Comment by eshellman** [comment]
@Viraj121 please address the comments

**Pull request closed without merging**